code/game/machinery/computer/pod.dm

```diff
@@ -304,14 +304,18 @@
 		visible_message("Cannot locate any mass driver of that ID. Cancelling firing sequence!")
 		return
 
-	if(teleporter_dest)
-		for(var/obj/structure/deathsquad_tele/D in world)
-			if(D.z != src.z)	continue
-			if(D.id_tag == ident_tag)
-				D.icon_state = "tele1"
-				D.ztarget = teleporter_dest
-				D.density = 1
-
+	var/spawn_marauder[] = new()
+	for(var/obj/effect/landmark/L in world)
+		if(L.name == "Marauder Entry")
+			spawn_marauder.Add(L)
+	for(var/obj/effect/landmark/L in world)
+		if(L.name == "Marauder Exit")
+			var/obj/effect/portal/P = new(L.loc)
```

Comment:
Same thing. Except now an invisible portal is used instead of a teleporter sprite. See https://github.com/ParadiseSS13/Paradise/blob/13f01232c98ca982d9f29f7deafddc25b0e2b13b/code/game/machinery/computer/specops_shuttle.dm#L144.